src/GraphODataTemplateWriter/CodeHelpers/Android/TypeHelperAndroid.cs

```diff
@@ -37,6 +37,13 @@ public static string GetTypeString(this OdcmType @type)
                     return "com.microsoft.graph.model.DateOnly";
                 case "Binary":
                     return "byte[]";
+                case "Stream":
```

Comment:
@iambmelt 
Android ignores streams at the property level and pushes developers to call a "getContent" method on the parent request builder. We aren't surfacing any property of type stream on the models.